### ISSUE TITLE
Time sync of radiant/dire teams upon reset not stable.

### ISSUE THREAD

**Issue by TimZaman** [TimZaman/dotaservice#33]
Ex (0.3.2):
```
ValueError: dota_time discrepancy in depleting initial worldstate queue.
at reset (/root/dotaservice/dotaservice/dotaservice.py:439)
at request_handler (/root/.local/lib/python3.7/site-packages/grpclib/server.py:327)
```

**Comment by sijiaxu** [comment]
I also keep getting this error when launch the game.
but, when I disable this part of code and use the GUI host mode, I stuck at the map loading screen and can not enter the game.
so is the time sync error cause this stuck problem?

**Comment by TimZaman** [comment]
Yeah that's exactly the problem.

**Comment by TimZaman** [comment]
Problem comes from what's explained in the last paragraph: https://github.com/TimZaman/dotaservice/blob/master/NOTES.md

**Comment by sijiaxu** [comment]
Thanks, but it seems cannot causing the stuck problem, I can not receive any normal observation(status='ok') in my client after launching the game.

**Comment by TimZaman** [comment]
Yep but that's because of this issue. Will try to solve tomorrow or
something

**Comment by sijiaxu** [comment]
another question, you mention set the render to true in order to see the GUI, is that referring to use the HOST_MODE_GUI mode?

**Comment by sijiaxu** [comment]
besides, can you upload your correctly running dota2 docker image or just a dota2 gz file to github, in this way I can check whether the difference in dota2 client causes my problem.

**Comment by lenLRX** [comment]
when i set ticks_per_observation to 1, i got ValueError: dota_time discrepancy in depleting initial worldstate queue.

when i set it to 2, i got assert queue.qsize() == 0 in observe()

It seems sync between two team caused a lot trouble.
But I wonder whether it is neccesery to keep sync. For me, i don't think so, i just want get the latest tick and it is ok to drop any data that i missed.
